storage/oasis/nodeapi/damask/node.go

```diff
@@ -27,48 +32,48 @@ import (
 // compatible with Damask gRPC API, this struct just trivially wraps the
 // convenience methods provided by oasis-core.
 type DamaskConsensusApiLite struct {
-	client consensus.ClientBackend
+	grpcConn *grpc.ClientConn
```

Comment:
will we use raw grpc for E* too?

Reply:
Yes, I decided to go that way in order to lose the asymmetry between versions. The use of oasis-core helper functions (as opposed to raw grpc) in Damask was to a large extent just inertia from back when we only supported a single version of oasis-core. In the end, those helper funcs don't benefit us all that much IMO: Code is a tiny bit more readable and shorter, but not much.